IanGRap/Squish
# Remixing

Contributors: Fernando, Alfredo 

Goal: Remix the audio to get a better balance between the different sounds

What changed: 
AudioMixer - was remixed
AudioManager.Prefab - updated some of the elements

How to test: 
* Open up Sprint 5 scene. Play the game. Set your computer volume to something comfortable (like if you were playing any other game). Go though the game and listen to the different components. 
* Try to make sure that none of the sounds were broken.
* Does everything sound ok? 

Note:  Balance in the audio is the most important part.
